exercism/bracket-push/bracket-push.js

```diff
@@ -0,0 +1,40 @@
+function bracket(str){
+var paren = ['(', ')']
+var curly = ['{','}']
+var square = ['[',']']
+var closings = [')', '}', ']']
```

Comment:
I might recommend using a single object to hold all these cases.

```javascript
let bracks = {
  paren: [ '(', ')' ],
  curly: [ '{', '}' ],
  square: [ '[', ']' ],
}
```

Then `brackets.paren` would give you `['(', ')']`, and `brackets.paren[0]` would give you `'('`.  

You might even just use the open brackets as the key and the closing brackets as the value.  `let obj = {'(': ')'}`.